### PR TITLE
Added Velocity Units

### PR DESCRIPTION
**Pull request by cpave3**
- Added Velocity Unit type
- added some common velocity units
- added unit tests for all current velocity units

### CODE REVIEW

File: src/Unit/Velocity/KilometrePerHour.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+namespace PhpUnitConversion\Unit\Velocity;
+
+use PhpUnitConversion\System\Metric;
+use PhpUnitConversion\Unit\Velocity;
+
+class KilometrePerHour extends Velocity implements Metric
```
Comment (pimlie) [inline src/Unit/Velocity/KilometrePerHour.php]: Please use `Meter` instead of `metre`

File: src/Unit/Velocity/MilePerHour.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace PhpUnitConversion\Unit\Velocity;
+
+use PhpUnitConversion\System\Imperial;
+use PhpUnitConversion\System\USC;
+use PhpUnitConversion\Unit\Velocity;
+
+class MilePerHour extends Velocity implements Imperial, USC
```
Comment (pimlie) [inline src/Unit/Velocity/MilePerHour.php:8]: According to wikipedia the official unit is `Miles per hour`? (https://en.wikipedia.org/wiki/Miles_per_hour)

Strange as it seems its Kilometer per hour (singular) but Miles per hour (plural)

File: src/Unit/Velocity/MilePerMinute.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace PhpUnitConversion\Unit\Velocity;
+
+use PhpUnitConversion\System\Imperial;
+use PhpUnitConversion\System\USC;
+use PhpUnitConversion\Unit\Velocity;
+
+class MilePerMinute extends Velocity implements Imperial, USC
```
Comment (pimlie) [inline src/Unit/Velocity/MilePerMinute.php:8]: Idem probably

File: src/Unit/Velocity/MilePerSecond.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace PhpUnitConversion\Unit\Velocity;
+
+use PhpUnitConversion\System\Imperial;
+use PhpUnitConversion\System\USC;
+use PhpUnitConversion\Unit\Velocity;
+
+class MilePerSecond extends Velocity implements Imperial, USC
```
Comment (pimlie) [inline src/Unit/Velocity/MilePerSecond.php:8]: Idem probably

File: tests/Unit/VelocityTest.php [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        $volumeUnit = new Velocity\Knot(100000);
+        $this->assertEquals('51444 m/s', Velocity\MeterPerSecond::from($volumeUnit)->format(0));
+    }
+
+    public function testFromMilesPerMinute()
+    {
+        $volumeUnit = new Velocity\MilePerMinute(100000);
+        $this->assertEquals('2682240.00 m/s', Velocity\MeterPerSecond::from($volumeUnit)->format(2));
+    }
+
```
Comment (pimlie) [inline tests/Unit/VelocityTest.php]: I think travis / code style complains about this empty line. Not sure

**Review by pimlie — COMMENTED** [review]
Hi, thanks for this PR!

Please see my comments, but looks great overall!

File: src/Unit/Velocity/MilePerMinute.php (hunk 3 above)
Comment (pimlie) [reply src/Unit/Velocity/MilePerMinute.php:8]: Never mind

File: src/Unit/Velocity/MilePerHour.php (hunk 2 above)
Comment (pimlie) [reply src/Unit/Velocity/MilePerHour.php:8]: Thinking about it a bit more I think its beter to keep this as MilePerHour but add a second class for MilesPerHour which only extends MilePerHour. That way we have the class name which is a combination of two other unit classes but also the official unit

File: src/Unit/Velocity/MilePerSecond.php (hunk 4 above)
Comment (pimlie) [reply src/Unit/Velocity/MilePerSecond.php:8]: Never mind

File: src/Unit/Velocity/KilometrePerHour.php (hunk 1 above)
Comment (cpave3) [reply src/Unit/Velocity/KilometrePerHour.php]: Done.

File: src/Unit/Velocity/MilePerHour.php (hunk 2 above)
Comment (cpave3) [reply src/Unit/Velocity/MilePerHour.php:8]: Done.

File: tests/Unit/VelocityTest.php (hunk 5 above)
Comment (cpave3) [reply tests/Unit/VelocityTest.php]: I removed the blank line, so hopefully that will satisfy Travis.

**Comment by pimlie** [comment]
Thanks, could you have a look at why the unit tests fail in php 7.2?

**Comment by cpave3** [comment]
Sorry about the delay. I have fixed the issue and all language versions are now passing.

**Review by pimlie — APPROVED** [review]

**Pull request merged**